src/utils/safeCommon.ts

```diff
@@ -1,8 +1,12 @@
+import { memo } from 'react';
+
 import {
     isDefined,
     isTruthyString,
 } from '@togglecorp/fujs';
 
+export const genericMemo: (<T>(c: T) => T) = memo;
```

Comment:
I don't understand this. Explain please?

Reply:
TS issues will occur if the memo is directly used in components that use [typescript generics](https://www.typescriptlang.org/docs/handbook/2/generics.html). This is done to make it type safe.
@tnagorra could you add more details?

Reply:
I could if I knew better!